src/utils.jl

```diff
@@ -9,6 +9,10 @@ end
 
 Base.show(io::IO, e::ChemfilesError) = show(io, "Chemfiles error: $(e.message)")
 
+"""
+If there was an error try to get it and throw a ChemfilesError with it.
+Show \"Unknown error\" in case it can't get the last one.
+"""
 function check(result::Integer, message = "Unknown error")
```

Comment:
This is actually a private function too. The `result` parameter is the `chfl_status` error code returned by the C library.